lib/Parser/Parse.cpp

```diff
@@ -9699,20 +9699,26 @@ ParseNodePtr Parser::ParseStatement()
         break;
 
     case tkID:
+    case tkLET:
```

Comment:
Does this not also apply to const?

Reply:
No, it seems that test262 and web reality only apply this this kind of behavior for 'let'. Having const after a label will be a syntax error regardless of if a new line comes after it.

Reply:
How about other context sensitive pseudo keywords? some interesting cases are hinted at in https://tc39.github.io/ecma262/#prod-Keyword e.g. yield/await/static.

Reply:
Is that (the expected const behavior) to spec?

Reply:
spec says that const may not be used as an identifier https://tc39.github.io/ecma262/#sec-keywords which is how it would be viewed as in `label: const \n x = 0`

Reply:
Added tests for yield/await and static